docker-compose.yml

```diff
@@ -283,6 +283,34 @@ services:
       - .:/app
       - bundle-3.1:/usr/local/bundle
       - "ddagent_var_run:${TEST_DDAGENT_VAR_RUN}"
+  tracer-3.1-arm64:
```

Comment:
`arm64` is not a good name, it doesn't even exist in ARM specs and is easily confused with `amd64`, it's only used by Darwin and qemu (and IIRC `aarch64` also works for qemu). I'd rather use `aarch64` which is in the ruby platform name, gcc target, and clang triplet.

Reply:
There's inconsistencies everywhere. Apple actually uses it on macOS: https://developer.apple.com/forums/thread/668206 .

If you feel strongly, I don't mind changing it, but I think "arm 64" is clearer than "aarch64".